apps/web/src/design-system/navigation/NavMenu.cy.tsx

```diff
@@ -0,0 +1,47 @@
+import { NavMenu } from './NavMenu';
+import { MemoryRouter as Router } from 'react-router-dom';
+import { TestWrapper } from '../../testing';
+
+it('should active when clicked menu item', () => {
```

Comment:
Please also add a describe for the test suite surrounding the test you implemented.
```
describe('NavMenu', () => {
...
});
```